### PR TITLE
chnaged clock format to HH:MM:SS

### PR DESCRIPTION
**Pull request by ciciofthe50**
the reset button no longer works properly so it needs to be fixed

### CODE REVIEW

File: code/CrockerClockFrame.java [hunk 1]
```diff
@@ -117,7 +121,14 @@ public void run() {
 						time = 0;
 						ClockDown.cancel(); // stops clock
 					} else {
-						clocky.setText(""+time);//puts word down
+						timeInSecond = time%60; //time in seconds
```
Comment (t-t-bui) [inline code/CrockerClockFrame.java:124]: Is this redundant?

You already set the timer values at the top. Is this to update the values?

**Review by t-t-bui — CHANGES_REQUESTED** [review]
take a look at the code. There is some redundancy in the code. See if you can clean it up otherwise code looks good.

File: code/CrockerClockFrame.java (hunk 1 above)
Comment (ciciofthe50) [reply code/CrockerClockFrame.java:124]: any redundancy in the action listen method should be ignored for now as the method does not work as intended I was planning to fix that in my next ticket

**Review by t-t-bui — CHANGES_REQUESTED** [review]
line 109 see if you can move out or give me a reason why it needs to be here.

**Comment by ciciofthe50** [comment]
https://www.youtube.com/watch?v=QEF62Fm81h4 @ 1:52

**Review by t-t-bui — APPROVED** [review]

**Pull request merged**